test/accelerators/NNPA/backend/CMakeLists.txt

```diff
@@ -434,6 +438,11 @@ if (ONNX_MLIR_ENABLE_JNI)
   add_dependencies(check-onnx-backend-nnpa-constant-jni PyRuntime)
   add_dependencies(check-onnx-backend-nnpa-constant-jni javaruntime)
   add_dependencies(check-onnx-backend-nnpa-constant-jni jniruntime)
+
+  add_dependencies(check-onnx-backend-numerical-nnpa check-onnx-backend-nnpa-jni)
+  add_dependencies(check-onnx-backend-numerical-nnpa check-onnx-backend-nnpa-dynamic-jni)
+  add_dependencies(check-onnx-backend-numerical-nnpa check-onnx-backend-nnpa-constant-jni)
```

Comment:
What is the naming convention here? `check-onnx-backend-nnpa-constant-jni` vs `check-onnx-backend-numerical-nnpa`, `nnpa` is at the end or  just after `backend`? I would prefer `check-onnx-backend-numerical-nnpa` to make them similar to CPU backend tests.

Reply:
Done. Thanks!